### PR TITLE
Bump bundlelib version

### PR DESCRIPTION
**Pull request by makyo**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui-charm/119/
Test PASSed.

File: server/guiserver/tests/bundles/test_views.py [hunk 1]
```diff
@@ -609,21 +609,26 @@ def test_valid_yaml(self):
...
-                     'method': 'addUnit'},
-                ],
-            },
+                    {
+                        u'args': ['cs:trusty/django-42'],
+                        u'requires': [],
+                        u'id': u'addCharm-0',
+                        u'method': u'addCharm'
+                    }, {
+                        u'args': [u'$addCharm-0', 'django', {}, u'', {}],
```
Comment (frankban) [inline server/guiserver/tests/bundles/test_views.py]: Could you please make sure the GUI already handles these two new deploy arguments?

File: server/guiserver/tests/bundles/test_views.py [hunk 2]
```diff
@@ -609,21 +609,26 @@ def test_valid_yaml(self):
...
+                    }, {
+                        u'args': [u'$deploy-1', None],
+                        u'requires': [u'deploy-1'],
+                        u'id': u'addUnit-2',
+                        u'method': u'addUnit'
+                    }
+                ]
+            }
         }
+        self.maxDiff = None
```
Comment (frankban) [inline server/guiserver/tests/bundles/test_views.py]: Remove? [shrug]

**Comment by frankban** [comment]
:+1: with a request and a trivial change. Thank you!

**Comment by bac** [comment]
👍

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui-charm/120/
Test PASSed.

**Comment by makyo** [comment]
Thanks all :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-charm-merge

**Pull request merged**